util/file.py

```diff
@@ -0,0 +1,29 @@
+"""
+Writing, loading, and opening files.
+"""
+import os
+import subprocess
+import platform
+
+
+def file_load(file: os.PathLike):
+    """Loads *file* and returns the contents as a string."""
+    with open(file, 'r') as f:
```

Comment:
might want to set the same encoding on load and save

Reply:
Duely noted. Will do testing. Prefer to see an error before fix.